templates/yamaha_show_environment.textfsm

```diff
@@ -0,0 +1,42 @@
+Value HARDWARE (\S+)
+Value VERSION (.+?)
+Value SERIAL (\S+)
+Value DATE (\d+/\d+/\d+)
```

Comment:
Can you namespace the date, time, etc. with either `CURRENT_X` or `BOOT_X`? It looks like they're capturing current, but in case someone adds to this in the future, the different times, etc. have the correct context.